cis/plugins/validation/schema.json

```diff
@@ -271,6 +271,13 @@
             "type": "string",
             "description": "Shirt size for this user. Format undefined."
         },
+        "tags": {
+            "type": "array",
+            "description": "General information about users tags. Tags prefixed by a provider name are provided by a specific tag origin.\nFor example `providername_l10n` is provided by `providername`.",
```

Comment:
we call them 'publisher' everywhere else so i'd call it `publishername_` here and elsewhere in the schema (oversight)
we can fix that as a diff pr though, ill open an issue